### PR TITLE
feat: quick links on playlist filters

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Filtering was confusing for people based on recent user interviews. We want to test offering a "simple filters" UI to help people get to more relevant recordings sooner.

## Changes

- Add simple filtering options behind [an experiment](https://app.posthog.com/experiments/3940). Adds an implicit "$pageview" event when using the 'Current URL' property
- Prevent someone from switching back to simple if advanced filters have been set
- Allow users to hide viewed recordings

Screenshot 2023-08-02 at 11 07 18
Screenshot 2023-08-02 at 11 06 04

https://github.com/PostHog/posthog/assets/6685876/31caf9e9-6231-4370-85cd-9fd9bc86cb27

## How did you test this code?

By 👁️ and snapshots. Will follow up with logic tests if experiment proves successful

### CODE REVIEW

File: frontend/src/scenes/session-recordings/filters/AdvancedSessionRecordingsFilters.tsx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+import { LemonLabel } from 'lib/lemon-ui/LemonLabel/LemonLabel'
+import { EntityTypes, FilterType, FilterableLogLevel, RecordingDurationFilter, RecordingFilters } from '~/types'
+import { DateFilter } from 'lib/components/DateFilter/DateFilter'
+import { DurationFilter } from './DurationFilter'
+import { LemonButtonWithDropdown, LemonCheckbox } from '@posthog/lemon-ui'
+import { TestAccountFilter } from 'scenes/insights/filters/TestAccountFilter'
+import { FlaggedFeature } from 'lib/components/FlaggedFeature'
+import { FEATURE_FLAGS } from 'lib/constants'
+
+export const AdvancedSessionRecordingsFilters = ({
```
Comment (pauldambra) [inline frontend/src/scenes/session-recordings/filters/AdvancedSessionRecordingsFilters.tsx:15]: I'll assume this is all moved unchanged and can be ignored 🙈

File: frontend/__snapshots__/scenes-app-recordings--recordings-list.png
Comment (pauldambra) [inline frontend/__snapshots__/scenes-app-recordings--recordings-list.png:1]: I really love visual regression tests 😊

**Review by pauldambra — APPROVED** [review]
This works really well! 🙌 

* Need to get it behind a feature flag so we can experiment with it?

File: frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    filters: RecordingFilters
+    setFilters: (filters: RecordingFilters) => void
+    localFilters: FilterType
+    setLocalFilters: (localFilters: FilterType) => void
+}): JSX.Element => {
+    const { currentTeam } = useValues(teamLogic)
+
+    const personPropertyOptions = useMemo(() => {
+        const properties = [{ label: 'Country', key: '$geoip_country_name' }]
+        const displayNameProperties = currentTeam?.person_display_name_properties ?? []
```
Comment (pauldambra) [inline frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx:35]: Ah! This doesn't work with the defaults

If you've never chosen one then we show a list of person props in settings even though the "setting" is empty.

Then if you re-order the default list we set it to a long list of options

We should probably take just the top one...

**Review by pauldambra — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/session-recordings/filters/AdvancedSessionRecordingsFilters.tsx (hunk 1 above)
Comment (daibhin) [reply frontend/src/scenes/session-recordings/filters/AdvancedSessionRecordingsFilters.tsx:15]: Yep! Just pulled it into a component

File: frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx (hunk 2 above)
Comment (daibhin) [reply frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx:35]: Great point. I think we have space for two so going to take the first two (if they exist)

**Review by pauldambra — APPROVED** [review]
🙌

**Review by pauldambra — APPROVED** [review]

**Pull request merged**